### ISSUE TITLE
Export table as excel

### ISSUE THREAD

**Issue by shannonpileggi** [rstudio/gt#240]
Feature request: Would it be possible and in the scope of this package to create an `as_excel()` function to export a gt table to excel?

**Comment by rich-iannone** [comment]
Thanks for posting this feature request. It is definitely within scope and this dovetails nicely with the feature request for Word table outputs. If the code generation required for these two things is similar enough, then this would be part of the same sub-project. That said, it may take awhile to begin work on this.

One question, do you want a function to take a data frame / tibble and simply create that in a new Excel document? Do you want all the footnotes and the header elements to be part of the Excel output? One thing we lose when doing all of the gt formatting is the cell type (everything gets converted to character) although I'm thinking that we want to preserve cell types for this output format.

**Comment by shannonpileggi** [comment]
That's great to hear!  I'd want a function that converts the gt table to an excel spread sheet.  And yes, I'd want to preserve the cell types as well.

Of particular interest to me: below is a screenshot of a standard deliverable for project teams with my employer, that I'd like to be able to fully replicate with R.

<img width="588" alt="standard_excel_table" src="https://user-images.githubusercontent.com/17747575/55651385-6879fc00-57b6-11e9-8e10-fbc9e102150d.PNG">

**Comment by berghe01** [comment]
Thanks for this amazing package! I too would really find an export to excel highly useful for converting these incredible gt_tables to publication/submission ready tables (often, journals request tables in excel format).

**Comment by jacobjchristensen** [comment]
I just want to add myself to the list of users that would love to see Excel export functionality. Would save researchers like myself a huge amount of time prepping tables prior to submitting to a scientific journal. dplyr, tidyr and friends can only get you so far, and whenever you change something, well, you know how the story goes...

**Comment by leebunce** [comment]
Agree with this, the ability to save tables to excel would be very helpful. As a government statistician it is common to release _multiple_ tables in a single excel document, as users are most comfortable with excel. An example is attached.
[Tables.xlsx](https://github.com/rstudio/gt/files/4469756/Tables.xlsx)

**Comment by ballengerj** [comment]
I agree as well, as I'm now looking for an easy way to export the table to Excel and or Word for Incorporation into a report.

**Comment by egoipse** [comment]
Hi... Here's a +1 for the possibility to export gtables to Excel and Word...

**Comment by rwaaijman** [comment]
Indeed, many thanks for this great package!!

Also wondering if there are any updates on this issue?
I would like to add gt tables to a tab in excel

**Comment by sethct** [comment]
Love this package and just wanted to add a +1 for excel exporting and whether this feature was implemented?

**Comment by aito123** [comment]
Hello, if its not too late i also want to add a +1 for this `as_excel` function. By the way, is it possible to rescale a gt table in a pdf  so that it feats all the horizontal page (it is a really wide table). Thanks!

**Comment by mbedan** [comment]
+1 for an Excel exporting capability

**Comment by ruampimentel** [comment]
I love this package!
Just want to add my +1 for this point. 
Any updates on this issue?

**Comment by LoganCones** [comment]
+1 on the exporting capabilities that would be great!

**Comment by Shelmith-Kariuki** [comment]
Hi ... thank you so much for this package. I am also hopeful that we can soon have a way to export our beautiful {gt} tables to excel.

**Comment by aawaa** [comment]
I am also looking to GT to Excel spreadsheet export! Thank you.

**Comment by matanhakim** [comment]
Amazing package, just started using it lately.
I would like to join the requests, right now my solution is to knit the {gt} table as flex_table and to copy it by hand from MS Word to Excel.
Thanks a lot for all of your work!

**Comment by BJ-Cochrane** [comment]
+1 to having Excel export functionality! Thanks to all devs!

**Comment by rshinytools** [comment]
+1

**Comment by nev-awaken** [comment]
+1

**Comment by vikasrawal** [comment]
+1

**Comment by blechturm** [comment]
Oh I absolutely would love this feature! +1 from me!

**Comment by cmwakibia** [comment]
Hi....I really love the gt library. The feature to export the results to excel as requested would be great !!!

**Comment by Shelmith-Kariuki** [comment]
In his talk at Posit::Conf(2024), @rich-iannone just mentioned that they are (trying to) working on Excel output tables. Thanks a lot 💃

![20240813_211201](https://github.com/user-attachments/assets/d5bf9811-cc06-4acd-af20-243f1d91875a)

**Comment by jkylearmstrong** [comment]
This would be a very helpful addition.

**Comment by jfgomezok** [comment]
Dear gt beloved community, there is now an alternative (not mine, all the credit to the author): https://github.com/JanMarvin/gtxlsx
